BrewMesModulesParent/common/src/main/java/com/brewmes/common/entities/Connection.java

```diff
@@ -15,6 +15,8 @@
     @NotNull
     private String name;
 
+    private boolean autobrewing = false;
```

Comment:
Set this in the constructor instead

Reply:
Not if it always has to default to false?